R/mail.R

```diff
@@ -0,0 +1,109 @@
+# Kaiaulu - https://github.com/sailuh/kaiaulu
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+
+
+
+#' Creates fake mbox reply data
+#'
+#' Used to create a fake mbox replies with the specified parameters and returns the output to the console. Can be used
+#' in conjunction with create_mbox_from_replies function to create a .mbox file to create a full fake .mbox file with
+#' multiple of these fake replies
+#'
+#' @param mlist specific mailing list associated with this mbox data
+#' @param reply_from_author name of sender. Formatted as: FIRSTNAME LASTNAME
+#' @param reply_from_email email of sender, automatically encapsulated in <> eg: sender@domain.com
+#' @param reply_to_author name of receiver. Formatted as: FIRSTNAME LASTNAME
+#' @param reply_to_email email of receiver, automatically encapsulated in <>  eg: receiver@domain.com
+#' @param reply_cc_author name of carbon copy of email to additional recipients. Formatted as: FIRSTNAME LASTNAME
+#' @param reply_cc_email email of carbon copy recipients, automatically encapsulated in <> input eg: recipientCC@domain.org
+#' @param reply_datetime Date, time in the following format: 2023-02-11T09:30:00  Be sure to add the capital T to separate date and time
+#' @param timezone  The abbreviation of desired timezone eg: HST
+#' @param reply_subject subject of the email as a string
+#' @param reply_body the body of the email as a string
+#' @return the content of the fake mbox you created
+#' @export
+create_fake_mbox_replies <- function(mlist, reply_from_author, reply_from_email, reply_to_author, reply_to_email, reply_cc_author, reply_cc_email, reply_datetime, timezone, reply_subject, reply_body) {
```

Comment:
Let's call this function `create_fake_mbox_reply` instead of plural. This function only generates a single reply at a time afterall.